File: src/main/java/bot/commands/Receiver.java
package bot.commands;

import bot.run.AbitVTBot;

public class Receiver {

    private final AbitVTBot bot;

    public Receiver(AbitVTBot bot) {
        this.bot = bot;
    }

    public void setCommands(Invoker invoker){
        invoker.add("/help", new HelpCommand(bot));
        invoker.add("/faq", new FaqCommand(bot));
        invoker.add("/subjects", new SubjectsCommand(bot));
        invoker.add("/links", new LinksCommand(bot));
        invoker.add("/plan", new PlanCommand(bot));
        invoker.add("/author", new AuthorCommand(bot));
        invoker.add("/settings", new SettingsCommand(bot));
        invoker.add("/start", new StartCommand(bot));
    }
}

File: src/main/java/bot/commands/Receiver.kt
package bot.commands

import bot.run.AbitVTBot

class Receiver(private val bot : AbitVTBot) {
    fun setCommands(invoker: Invoker) {
        invoker.add("/help", HelpCommand(bot))
        invoker.add("/faq", FaqCommand(bot))
        invoker.add("/subjects", SubjectsCommand(bot))
        invoker.add("/links", LinksCommand(bot))
        invoker.add("/plan", PlanCommand(bot))
        invoker.add("/author", AuthorCommand(bot))
        invoker.add("/settings", SettingsCommand(bot))
        invoker.add("/start", StartCommand(bot))
    }
}